_layouts/redirect.html

```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+      <meta charset="utf-8"/>
+      <meta http-equiv="refresh" content="1;url={{ page.redirect }}"/>
+      <link rel="canonical" href="{{ page.redirect }}"/>
+      <script type="text/javascript">
+              window.location.href = "{{ page.redirect }}"
+      </script>
+      <title>Page Redirection</title>
+  </head>
+  <body>
+  If you are not redirected automatically, follow <a href='{{ page.redirect }}'>this link</a>.
```

Comment:
Maybe add a note here: 

"The content has moved. If you are not redirected automatically, ..."

Reply:
done